webapp/src/components/ProfilePage/ProfilePage.container.js

```diff
@@ -42,6 +43,13 @@ const mapState = (state, { location, match }) => {
   }
 
   const publishedAssets = publishedParcels.concat(publishedEstates)
+  const archivedBids = getArchivedBids(state)
+
+  if (tab === PROFILE_PAGE_TABS.archivebids) {
+    bids = bids.filter(bid => archivedBids[bid.id])
+  } else {
+    bids = bids.filter(bid => !archivedBids[bid.id])
+  }
```

Comment:
shouldn't this be a part of the switch below?

the true condition should be on `case PROFILE_PAGE_TABS.archivebids:`

and the false on `case PROFILE_PAGE_TABS.bids:`

I don't think it'd be a problem with the tab count because it's changing at the moment (it shows the archived/unarchived count)

Reply:
If you place that as part of the switch, you will see a count with all the bids.

When you are in another tab (not `bid`), you want to see the number of bids that you will see once you enter in that tab. And, as archived ones are hidden, the correct number to show is the unarchive bids.